### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShadowMonster99**
Removed
- Reference to SpaceTheme for discord as users have gotten confused and tried to install it from there. 
- Version info (Already on the left-most panel)
- Discord Server (Already on the left-most panel)
- Kofi Support Link (Already on the left-most panel)

Added
- A proper description explaining how to install the theme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,18 +12,9 @@
...
-
-**6.** Go back to Steam Settings, select Interface, and choose the SpaceTheme from the dropdown menu.
-
-**7.** Finally, restart Steam, and you should have the SpaceTheme installed. Enjoy your new skin!
-
+1. Download the [Millennium](https://steambrew.app/). Follow the instructions provided to install the patcher.
+1. Open Steam and navigate to `Steam -> Settings -> Interface -> Open Millennium`
+1. With the Millennium interface open, drag the install button overtop of the interface. 
+1. From there, you will see a install prompt. Now simply drop the theme
+1. Select the theme from the interface after it installs.
```
Comment (SpaceEnergy) [inline README.md:19]: the listing numbers are not correct and maybe add a text at step 2 that you are not allowed to use Steam Beta
```suggestion
1. Download the [Millennium](https://steambrew.app/). Follow the instructions provided to install the patcher.
2. Open Steam and navigate to `Steam -> Settings -> Interface -> Open Millennium` _(Make sure you're not in the Steam beta)_
3. With the Millennium interface open, drag the install button overtop of the interface. 
4. From there, you will see a install prompt. Now simply drop the theme
5. Select the theme from the interface after it installs.
```

**Review by SpaceEnergy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ShadowMonster99) [reply README.md:19]: using 1. is the proper markdown syntax. it auto increments for you. As for the Steam Beta notice, it should be fine considering there is already a warning on the front page of the website. If we update it now, we will just have to update it again later!

File: README.md [hunk 2]
```diff
@@ -1,11 +1,5 @@
-[![Banner](https://i.imgur.com/2bQmNZB.png)](https://github.com/SpaceEnergy/SpaceTheme-Discord)
+# SpaceTheme for Steam
 
-# SpaceTheme for Steam | v1.0.4-alpha
-[Join Discord Server for Support, News and Updates](https://discord.gg/7Zv8Xz3Vzn)
-
-[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/spacetheme)
-
-## Preview v1.0.4-alpha
 ![Preview](https://i.imgur.com/YYSSkBy.png)
```
Comment (SpaceEnergy) [inline README.md:3]: change alt text to "v1.0.4-alpha preview"

**Review by SpaceEnergy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ShadowMonster99) [reply README.md:3]: merge the pull request and make further changes you see fit.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,5 @@
-[![Banner](https://i.imgur.com/2bQmNZB.png)](https://github.com/SpaceEnergy/SpaceTheme-Discord)
```
Comment (SpaceEnergy) [inline README.md:1]: Add the banner for the DiscordTheme at the bottom
```Suggested change
## Installation
1.
...

## Other Themes
[![Banner](https://i.imgur.com/2bQmNZB.png)](https://github.com/SpaceEnergy/SpaceTheme-Discord)
```

**Review by SpaceEnergy — CHANGES_REQUESTED** [review]

**Review by SpaceEnergy — APPROVED** [review]

**Pull request merged**